### PR TITLE
CocoaPods support

### PR DESCRIPTION
**Pull request by Brett-Best**
Added a starting pod spec, would be great if you could check it.
Pretty much the same format as the PureSwiftUI podspec.

This is dependent on https://github.com/CodeSlicing/pure-swift-ui/pull/1 getting merged.

### CODE REVIEW

**Comment by CodeSlicing** [comment]
Ready to approve this PR. Could you do a merge from develop since I've updated develop to the latest fixes for the segmentation fault. Cheers.

**Comment by CodeSlicing** [comment]
Actually I do have a concern regarding the spec in PureSwiftUITools. Since I'm not signed up in Trunk yet, it doesn't have native support for versioning so the pod spec would have to refer to a specific tag in the PureSwiftUI repository. 

I'm thinking at that until I get the libraries registered in Trunk we should hold off on committing a pod spec for this library. Happy to hear your thoughts on this, but I'd rather not be referring to manual tags in the pod spec for a required dependency.

**Comment by Brett-Best** [comment]
Hmmm, my Podfile looks like this:
```ruby
  pod 'PureSwiftUI', :git => 'https://github.com/CodeSlicing/pure-swift-ui.git', :tag => '1.15.0'
  pod 'PureSwiftUITools', :git => 'https://github.com/Brett-Best/pure-swift-ui-tools.git', :branch => 'develop'
```

I think if you merged develop into master you can drop the :tag / :branch.
The dependency resolution should still work based off of the library version in the Podspec (I had to remove the dependency version requirement I added in the pod spec to test the beta of PureSwiftUITools.

I’m not sure what the process of getting the libraries added to the trunk CDN is.

EDIT:
https://guides.cocoapods.org/making/getting-setup-with-trunk.html
Doesn’t look too hard.

File: PureSwiftUITools.podspec [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  spec.macos.deployment_target = "10.15"
+  spec.tvos.deployment_target = "13.0"
+  spec.watchos.deployment_target = "6.0"
+  
+  spec.source = { :git => "https://github.com/CodeSlicing/pure-swift-ui-tools.git", :tag => "#{spec.version}" }
+
+  spec.source_files  = "Sources/**/*.{swift}"
+  spec.swift_version = "5.1"
+  spec.framework  = "SwiftUI"
+  spec.dependency 'PureSwiftUI', '~> 3.0.0-beta-9'
```
Comment (CodeSlicing) [inline PureSwiftUITools.podspec:23]: Please update this to specifically point at the PureSwiftUI branch feature/3.0.0 - I don't trust beta versioning so let's go with a branch for now. Upon release I'll update to server tag up to next major. Cheers

**Comment by CodeSlicing** [comment]
Closing this. Trying to get on top of things again.

**Pull request closed without merging**